File: 권현욱/1주차/2562.cpp
#include <iostream>

using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int arr[10];
    for (int i = 0; i < 9; i++)
    {
        cin >> arr[i];

        if (arr[i] >= 100)
            return (0);
    }
    
    int max = 0;
    int idx = 0;

    for (int i = 0; i < 9; i++)
    {
        if (arr[i] > max)
        {
            max = arr[i];
            idx = i;
        }
    }

    cout << max << "\n" << idx + 1;

    return 0;
}

File: 고경남/1주차/2562.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner	sc = new Scanner(System.in);
		
		int arr[] = new int[9];
		for(int i=0; i<9; i++) {
			arr[i] = sc.nextInt(); 
		}
		
		
		int idx = 0;
		int max = 0;
		
		for(int i=0; i<9; i++) {
			if(arr[i] > max) {
				max = arr[i];
				idx = i;
			}
		}
		
		System.out.println(max);
		System.out.println(idx + 1);

	}
}
